### ISSUE TITLE
Problem using eb on POWER architecture

### ISSUE THREAD

**Issue by chwilk** [hpcugent/easybuild#32]
I'm getting a traceback from eb trying to run on a POWER8 machine running RHEL6.5.

````
Traceback (most recent call last):
  File "/opt/apps/python/lib/python2.6/site-packages/easybuild/main.py", line 477, in <module>
    main()
  File "/opt/apps/python/lib/python2.6/site-packages/easybuild/main.py", line 129, in main
    init_session_state = session_state()
  File "/opt/apps/python/lib/python2.6/site-packages/easybuild/tools/testing.py", line 156, in session_state
    'system_info': get_system_info(),
  File "/opt/apps/python/lib/python2.6/site-packages/easybuild/tools/systemtools.py", line 461, in get_system_info
    'cpu_model': get_cpu_model(),
  File "/opt/apps/python/lib/python2.6/site-packages/easybuild/tools/systemtools.py", line 199, in get_cpu_model
    return regexp.search(txt).groupdict()['modelname'].strip()
AttributeError: 'NoneType' object has no attribute 'groupdict'
````

A `head` of `/proc/cpuinfo` from the machine in question shows the trouble. There's no `modelname` line:

````
processor       : 0
cpu             : POWER7 (architected), altivec supported
clock           : 4116.000000MHz
revision        : 2.1 (pvr 004b 0201)

processor       : 1
cpu             : POWER7 (architected), altivec supported
clock           : 4116.000000MHz
revision        : 2.1 (pvr 004b 0201)
````

(Note that the P8 isn't fully supported by the RHEL6.5 kernel, so it shows up as a POWER7. This is expected. (My POWER7 cluster shows similar output).

**Comment by boegel** [comment]
unit test suite is now a lot more robust, now that https://github.com/hpcugent/easybuild-framework/pull/1167 is merged in

The necessary changes in https://github.com/hpcugent/easybuild-framework/issues/1044 to support Linux/POWER should be going in, in time for EasyBuild v2.0, soon, so I'm going to close this issue.

Thanks again for reporting this @chwilk !

**Comment by boegel** [comment]
fyi: https://github.com/hpcugent/easybuild-framework/pull/1044 was merged in, will be included in EasyBuild v2.0
